lib/Models/AugmentedVirtuality.ts

```diff
@@ -62,20 +62,7 @@ export default class AugmentedVirtuality {
   // (and increment) we cycle and go to the first height.
   @observable hoverLevel = AugmentedVirtuality.PRESET_HEIGHTS.length - 1;
 
-  constructor(readonly terria: Terria) {
-    // Always run the device orientation event, this way as soon as we enable we
-    // know where we are and set the orientation rather then having to wait for
-    // the next update.  The following is disabled because chrome does not
-    // currently support deviceorientationabsolute correctly: if
-    // ('ondeviceorientationabsolute' in window) {
-    // window.addEventListener('deviceorientationabsolute', function(event)
-    // {that._orientationUpdate(event);} ); } else
-    if ("ondeviceorientation" in window) {
-      window.addEventListener("deviceorientation", event =>
-        this.storeOrientation(event)
-      );
-    }
-  }
```

Comment:
Moved this to `startEventLoop` method to avoid unnecessary changes even when AR is not activated by the user.